### PR TITLE
Resolving cleanup issue on tiobench.py

### PR DESCRIPTION
**Pull request by Naresh-ibm**
script was not properly cleaning up while exiting
after each varient runs. so fixing it.

Signed-off-by: Naresh Bannoth <nbannoth@in.ibm.com>

### CODE REVIEW

**Comment by Naresh-ibm** [comment]
[complete-fc-bucket-run-logs.txt](https://github.com/avocado-framework-tests/avocado-misc-tests/files/9015654/complete-fc-bucket-run-logs.txt)

File: io/disk/tiobench.py [hunk 1]
```diff
@@ -41,6 +44,12 @@ class Tiobench(Test):
     Avocado test for tiobench.
     """
 
+    def clear_dmesg(self):
+        """
+        clears the dmesg logs.
+        """
+        process.run("dmesg -C ", sudo=True)
```
Comment (PraveenPenguin) [inline io/disk/tiobench.py:51]: please use avocado-based `dmesg` utility 

https://github.com/avocado-framework/avocado/blob/master/avocado/utils/dmesg.py

**Review by PraveenPenguin — CHANGES_REQUESTED** [review]

**Comment by PraveenPenguin** [comment]
@Naresh-ibm i know we discuss moving some of the raid utility in avocado have we changed the plan  , @abdhaleegit what you say ?

**Comment by abdhaleegit** [comment]
> @Naresh-ibm i know we discuss moving some of the raid utility in avocado have we changed the plan , @abdhaleegit what you say ?

it has already moved some of the raid utility

File: io/disk/tiobench.py (hunk 1 above)
Comment (Naresh-ibm) [reply io/disk/tiobench.py]: > please use avocado-based `dmesg` utility
> 
> https://github.com/avocado-framework/avocado/blob/master/avocado/utils/dmesg.py

Done. Thanks for pointing this.

**Review by abdhaleegit — APPROVED** [review]
LGTM

**Pull request merged**